server/webapp/WEB-INF/rails/webpack/views/pages/users/index.scss

```diff
@@ -206,7 +206,7 @@ $selected-table-row: #e2f7fa;
 
 .table-row {
   border-bottom: 1px solid $border-color;
-  height:        45px;
+  min-height:        45px;
```

Comment:
Not sure if the indentation is right.
#### Currently, it is:
![Screen Shot 2019-05-06 at 10 40 26 AM](https://user-images.githubusercontent.com/15275847/57207449-71473300-6feb-11e9-93ab-c11be2463450.png)


#### How about:
![Screen Shot 2019-05-06 at 10 40 37 AM](https://user-images.githubusercontent.com/15275847/57207446-6f7d6f80-6feb-11e9-9b32-bf4dc620c658.png)

?

---

Setting indentation settings of scss to align on values (in IntelliJ) will do the trick :)
![Screen Shot 2019-05-06 at 10 42 04 AM](https://user-images.githubusercontent.com/15275847/57207533-02b6a500-6fec-11e9-889f-732d3c673c07.png)